controllers/user/UserController.php

```diff
@@ -102,6 +103,17 @@ public function actionGetByAuthclient($name, $id)
         return $this->actionView($user->id);
     }
 
+    public function actionGetBySource($source, $id)
+    {
+        $auth = UserAuth::findOne(['source' => $source, 'source_id' => $id]);
```

Comment:
There is also some primary auth source in the `user` record. `user.authclient_id` (source_id)  and `auth_mode` (source). 

Maybe we should additionally try to find the user based on this table.

Reply:
@gevorgmansuryan Please see the example here:

https://github.com/humhub/humhub/blob/master/protected/humhub/modules/user/services/AuthClientService.php#L176-L182

We have some Primary AuthClient which is stored in the `user` table and some additional AuthClients (e.g. OAuths) in the `user_auth` table.

I think it would be better when this REST endpoint covers both sources.

Reply:
@luke- I think this will not work in this case as client mentioned that auth_mode is always “local” and
authclient_id is always null in their database